client/signup/steps/domains/style.scss

```diff
@@ -1,6 +1,6 @@
 .domains__step-section-wrapper {
 	margin: 0 auto;
-	max-width: 650px;
```

Comment:
This seemed too narrow for this new screen and I didn't want to have different sizes for the different sections. 720px seems to work fine for all of the sections in the domains step, but please let me know if I missed something here.